.env.reference

```
@@ -0,0 +1,455 @@
+#
+# NOTE: Many packages still receive their DIRECT configuration through explicit
+# Go structs (redis, mongo, rabbitmq, postgres, circuitbreaker, jwt, crypto,
+# license, server, backoff, errgroup, safe, transaction, cron, pointers,
+# constants, security, log, runtime, opentelemetry/metrics), but the global
+# security-policy env vars below can still influence their runtime behavior.
+#
+# Generated from source scan — 2026-03-23
+# Updated: 2026-03-24 — Added SECURITY_TIER override and fail-closed rate-limit notes
+# =============================================================================
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)